packages/web/services/github.ts

```diff
@@ -0,0 +1,27 @@
+import axios from "axios";
```

Comment:
How about moving this to packages/web/queries, where we typically store queries to be run on the server?

Reply:
Yes, it sounds good!

Reply:
Done https://github.com/osmosis-labs/osmosis-frontend/pull/2234/commits/a87650c9959226fc71828eabaa13c50d1bd75336